app/data/pencil2d.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>pencil2d</id>
```

Comment:
I wanted to submit another PR for the renaming, because it involves some other files too:

* app/data/pencil2d.desktop
* app/data/pencil2d.png
* app/data/pencil2d.xml